src/to-fresh.js

```diff
@@ -71,6 +71,7 @@ Convert JRS resume sections to FRESH.
             start: edu.startDate,
             end: edu.endDate,
             grade: edu.gpa,
+            title: edu.title,
```

Comment:
It's definitely a bit of a hack, but I think it's certainly better than using a field that doesn't exist. `title` certainly seems like a poorly defined field. `studyType + ' of ' + area` seems like it would produce something that could be a valid title, but there are definitely things that don't conform to that format that could also be valid titles.